configs/example.yaml

```diff
@@ -63,53 +64,67 @@ chemical_symbols:
   - C
 
 # logging
-wandb: true                                                                       # we recommend using wandb for logging, we'll turn it off here as it's optional
+wandb: true                                                                        # we recommend using wandb for logging
 wandb_project: toluene-example                                                     # project name used in wandb
 
-verbose: info                                                                      # the same as python logging, e.g. warning, info, debug, error. case insensitive
-log_batch_freq: 1000000                                                                  # batch frequency, how often to print training errors withinin the same epoch
+verbose: info                                                                      # the same as python logging, e.g. warning, info, debug, error; case insensitive
+log_batch_freq: 10                                                                 # batch frequency, how often to print training errors withinin the same epoch
 log_epoch_freq: 1                                                                  # epoch frequency, how often to print and save the model
-save_checkpoint_freq: -1                                                           # frequency to save the intermediate checkpoint. no saving when the value is not positive.
-save_ema_checkpoint_freq: -1                                                       # frequency to save the intermediate ema checkpoint. no saving when the value is not positive.
+save_checkpoint_freq: -1                                                           # frequency to save the intermediate checkpoint. no saving of intermediate checkpoints when the value is not positive.
```

Comment:
@nw13slx This should only affect intermediate checkpoint if negative right, best one is still saved.

Reply:
right.